pkg/codegen/python/doc.go

```diff
@@ -63,9 +62,11 @@ func (d DocLanguageHelper) GetLanguageTypeString(pkg *schema.Package, moduleName
 		switch dictionaryTy := t.(type) {
 		case *schema.MapType:
 			elType := dictionaryTy.ElementType.String()
-			return getDictWithTypeName(elementTypeToName(elType))
+			return getMapWithTypeName(elementTypeToName(elType))
 		case *schema.ObjectType:
 			return getDictWithTypeName(tokenToName(dictionaryTy.Token))
+		default:
+			return "Dict[Any, Any]"
```

Comment:
Shouldn't this be:

```suggestion
			return "Dict[str, Any]"
```

?

Reply:
Yes, absolutely! Based on your other comment, the key for this should always be of type string.